pwa-shell/src/styles/topbar.scss

```diff
@@ -1,6 +1,12 @@
 .topbar {
   background-color: $color-primary-lightest;
 
+  &.fixed-top {
```

Comment:
which issue does this solve?

Reply:
when the page is going to view the article, there is this involuntary movement of the logo going sideways, which i thought it may be a glitch because of the width of the height expands more than 100%. So I made it strict to make the header width fixed :)